README.md

```diff
@@ -10,3 +10,17 @@ Contains the debug adapter registry and adapter templates
     ┣ 📄 *.adapter.json       json template for vscode launch and task definitions per debug adapter  
     ┗ 📄 ...
 ```
+
+The adapter registry is used by CMSIS-Toolbox starting version 2.9.0 to generate `<solution>+<target-type>.cbuild-run.yml` files for the active target-set selected by 
+
+```
+cbuild setup <solution> --active <target-type>@<set>
```

Comment:
Align with cbuild documentation: target set is optional (may be unnamed)
```suggestion
cbuild setup <solution> --active <target-type>[@<set>]
```

Reply:
Good catch. Valid correction.